core/admin/mailu/configuration.py

```diff
@@ -54,6 +56,8 @@
     # Advanced settings
     'PASSWORD_SCHEME': 'PBKDF2',
     'LOG_LEVEL': 'WARNING',
+    'SESSION_KEY_BITS': 128,
```

Comment:
Hmm. I think I didn't document SESSION_KEY_BITS in docs/configuration.rst

Reply:
this is a "manual" backport to 1.8 ... I suggest that we approve it unless something is terribly off... and keep working on improvements through the "normal" channel: in a different PR against master (that we can also backport as required but using mergify)